samples/samples-js/AddProduct/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = async function (context) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const products = [{
+        "productid": 1001,
+        "name": "Shoes",
+        "cost": "100"
+    }, {
+        "productid": 1002,
+        "name": "Flowers",
+        "cost": "500"
+    }];
+
+    context.bindings.product = JSON.stringify(products);
```

Comment:
whatever is defined in the function.json in the bindings part becomes context.bindings. Since I have the name as product in there, I have to use that name to define data. I initially tested with one product, will update it.